spec/takeaway_spec.rb

```diff
@@ -0,0 +1,15 @@
+require_relative '../lib/takeaway'
+
+describe Takeaway do
+  subject(:takeaway) { described_class.new(menu: menu, order: order, sms: sms)}
```

Comment:
It looks like this test fails with a useful error message as to what is going wrong. Can you use this message to fix the error?